src/components/Header/UpgradePlanForm/UpgradePlanForm.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import axios from 'axios';
+
+class UpgradePlanForm extends React.Component {
```

Comment:
What do you think about moving this component in a place independent of _Header_? It seems to be only a form, so, it could be used independently of the header.

Also, it depends on Doppler Backend, in my opinion, it is another reason to have it in a more accessible place.

Reply:
Yes you right i can move in another folder